steps/python/python.release.yml

```diff
@@ -28,8 +28,6 @@ steps:
     versionSpec: "3.6"
     architecture: "x64"
 
-- template: python.deps.yml
```

Comment:
Probably it will not cause a build failure but maybe we should instead run `pip -r requirements_dev.txt` in the step below?

Reply:
Per discussion we had, moving setuptools and pip install to the "Install publish prereqs" step.